stdlib/_blake2.pyi

```diff
@@ -13,7 +14,7 @@ BLAKE2S_PERSON_SIZE: int = 8
 BLAKE2S_SALT_SIZE: int = 8
 
 @final
-class blake2b:
+class blake2b(HashObject):
```

Comment:
Nit: These classes don't need to inherit from the protocol, and deriving from it has no effect in stubs. So we should follow the implementation here.